policyengine_us/reforms/tax_exempt/tax_exempt_reform.py

```diff
@@ -43,7 +43,7 @@ def formula(person, period, parameters):
             if p.tip_income.payroll_tax_exempt:
                 exempt_income += person("tip_income", period)
             if p.overtime.payroll_tax_exempt:
-                exempt_income += person("overtime_income", period)
+                exempt_income += person("fsla_overtime_premium", period)
```

Comment:
Fine here but just a note to be super careful with `+=`. It has tripped us up in the past by adding super non-obvious behaviour, e.g. by side-effecting the original source of the variable.